packager/media/formats/mp2t/ts_section_pmt.cc

```diff
@@ -95,7 +96,29 @@ bool TsSectionPmt::ParsePsiSection(BitReader* bit_reader) {
 
     // Read the ES info descriptors.
     // Defined in section 2.6 of ISO-13818.
-    RCHECK(bit_reader->SkipBits(8 * es_info_length));
+    int descriptor_tag;
+    int descriptor_length;
+    char lang[] = { 'u', 'n', 'd', '\0' };
+
+    while (es_info_length) {
+	RCHECK(bit_reader->ReadBits(8, &descriptor_tag));
```

Comment:
Please replace tab with spaces. Same elsewhere. We do not use tab in this project as it does not render consistently on different systems.

You can check the format using `packager/tools/git/check_formatting.py`:

```bash
$ packager/tools/git/check_formatting.py <base>
```

<base> is the commit before your change. An example is HEAD^.